app/_hub/kong-inc/application-registration/index.md

```diff
@@ -122,11 +122,96 @@ params:
         should last, after which the client will need to refresh the token.
         Set to `0` to disable the expiration.
   extra: |
-    When configuring the plugin, at least one of the following OAuth2
-    auth flows must be enabled:
+    **Important:** When configuring the plugin, at least one of the following
+     OAuth2 auth flows must be enabled:
     - Authorization Code
     - Client Credentials
     - Implicit Grant
     - Password Grant
 
 ---
+
+## Examples
```

Comment:
I tested all of these on my local instance. There are some issues with the plugin template. Sometimes we need to deviate from the schema saying a field is required, such as for the provision_key, which is autogenerated.

Reply:
Another thing: the template forces use to enable all oauth flows (it is 1 or more required) but that is an unlikely, unrealistic use case for customers. The enable plugin portion of the template causes some schema errors the way the logic currently is not only in this plugin but others.